src/Symfony/Component/Serializer/Mapping/AttributeMetadataInterface.php

```diff
@@ -53,14 +53,33 @@ public function getMaxDepth();
 
     /**
      * Sets the serialization name for this attribute.
+     *
+     * @deprecated
      */
     public function setSerializedName(string $serializedName = null);
 
+    /**
+     * Add the serialization name for this attribute.
+     */
+    public function addSerializedName(string $serializedName = null, array $groups = []);
```

Comment:
Adding new methods on an interface is not possible (BC break). You should add them as `@method` annotations on the class instead.

Reply:
Your Symfony BC documentation defines that @internal interface can have BC, no ?
https://symfony.com/doc/current/contributing/code/bc.html

Reply:
You're right, I didn't see that it was internal, all is good.